axelrod/strategies/oncebitten.py

```diff
@@ -8,7 +8,13 @@ class OnceBitten(Player):
     """
 
     name = 'Once Bitten'
-    memory_depth = float('inf')  # Long memory
+    behaviour = {
+        'memory_depth': float('inf'),  # Long memory
```

Comment:
Again perhaps this one is memory_depth =12 (10 rounds of altered behavior following two rounds of opponent defection)